config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Expertiza::Application.routes.draw do
+  resources :grading_histories
```

Comment:
I'd recommend moving this to the appropriate line in this file so as to maintain alphabetical order of routes.